lib/box/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+
+var box = angular.module('guBox', []);
+
+box.directive('guBox', function boxDirective(){
+  return {
+    restrict: 'AE',
```

Comment:
Would it be possible to add some logic in the `link` function to only allow these two options and not any of the other ones above?